app/uk/gov/hmrc/exports/controllers/DeclarationController.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.libs.json.Json
+import play.api.mvc._
+import uk.gov.hmrc.exports.models.declaration.ExportsDeclaration
+import uk.gov.hmrc.exports.services.DeclarationService
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@Singleton
+class DeclarationController @Inject()(
+  declarationService: DeclarationService,
+  override val controllerComponents: ControllerComponents
+) extends RESTController {
+
+  def post(): Action[ExportsDeclaration] = Action.async(parsingJson[ExportsDeclaration]) { implicit request =>
```

Comment:
Do we think this needs any kind of Auth?

Reply:
I think so - it even describe in platform patterns - [Assumed Upstream Security](https://confluence.tools.tax.service.gov.uk/display/AR/Assumed+Upstream+Security)

Reply:
Auth Added